### DISCUSSION TITLE
How to use command mode in status bar?

### DISCUSSION THREAD

**Discussion by andynameistaken** [vscode-neovim/vscode-neovim#1768] [Q&A]
Is it possible to see commands in status bar like in VscodeVim?

![Screenshot 2024-01-25 at 21 28 23](https://github.com/vscode-neovim/vscode-neovim/assets/15692002/ce40bcf3-572e-42b6-8fcf-e51de00ecf9f)

**Comment by gaoqiangks** [comment]
`set showcmd`

![image](https://github.com/vscode-neovim/vscode-neovim/assets/52161883/84e816ba-9cb8-44f4-a049-4612066e0514)
